includes/wp-admin/class-health-check.php

```diff
@@ -432,6 +432,18 @@ public static function test_pretty_permalinks() {
 					esc_url( admin_url( 'options-permalink.php' ) )
 				)
 			);
+		} elseif ( str_starts_with( $permalink_structure, '/index.php' ) ) {
+			$result['status']         = 'critical';
+			$result['label']          = \__( 'ActivityPub does not support the "Almost Pretty Permalinks" structure.', 'activitypub' );
+			$result['badge']['color'] = 'red';
+			$result['description']    = \sprintf(
+				'<p>%s</p>',
+				sprintf(
+					/* translators: %s: Permalink settings URL. */
+					\__( 'ActivityPub needs SEO-friendly URLs to work properly. Please <a href="%s">remove the <code>/index.php</code></a> from your permalink structure.', 'activitypub' ),
```

Comment:
AI suggestion: "Your current permalink structure includes "/index.php" which is not compatible with ActivityPub. Please <a href="%s">update your permalink settings</a> to use a standard format without "/index.php"."